android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -36,6 +36,7 @@
 import android.content.Context;
 import android.os.SystemClock;
 import android.util.DisplayMetrics;
+import android.util.Log;
```

Comment:
@kostyaBoss You can remove the unused import `import android.util.Log;` on line 39.